var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -159,6 +159,8 @@ class Dealii(CMakePackage, CudaPackage):
     depends_on('ginkgo',           when='@9.1:+ginkgo')
     depends_on('gmsh+tetgen+netgen+oce', when='@9.0:+gmsh', type=('build', 'run'))
     depends_on('gsl',              when='@8.5.0:+gsl')
+    # FIXME: next line fixes concretization with trilinos and adol-c
+    depends_on('trilinos~exodus~netcdf',    when='@9.0:+adol-c')
```

Comment:
Makes sense, due to the added the conflicts. But shouldn't this be `when='@9.0:+adol-c+trilinos'`? 

Also, could we please put this further down the list (~line 199 with the other trilinos depends)? That way this list remains (nearly) alphabetically ordered.

Reply:
Thanks for the fix, BTW :-)

Reply:
Thanks @jppelteret!